packages/webgal/src/Core/gameScripts/changeFigure.ts

```diff
@@ -39,12 +54,49 @@ export const changeFigure = (sentence: ISentence): IPerform => {
     if (e.key === 'motion') {
       motion = e.value.toString();
     }
+    if (e.key === 'mouthOpen') {
+      mouthOpen = e.value.toString();
+      mouthOpen = assetSetter(mouthOpen,fileType.figure);
+    }
+    if (e.key === 'mouthClose') {
+      mouthClose = e.value.toString();
+      mouthClose = assetSetter(mouthClose,fileType.figure);
+    }
+    if (e.key === 'mouthHalfOpen') {
+      mouthHalfOpen = e.value.toString();
+      mouthHalfOpen = assetSetter(mouthHalfOpen,fileType.figure);
+    }
+    if (e.key === 'eyesOpen') {
+      eyesOpen = e.value.toString();
+      eyesOpen = assetSetter(eyesOpen,fileType.figure);
+    }
+    if (e.key === 'eyesClose') {
+      eyesClose = e.value.toString();
+      eyesClose = assetSetter(eyesClose,fileType.figure);
+    }
+    if (e.key === 'animationFlag') {
+      animationFlag = e.value.toString();
+    }
     if (content === 'none') {
       content = '';
     }
   }
-  const dispatch = webgalStore.dispatch;
-
+  if(animationFlag){
```

Comment:
When the illustration is switched or unloaded, the mouth shape or blink animation associated with that key should be deleted.
In addition, the handling here is strange. Shouldn't it be to first retrieve the previous array, make some modifications, and then dispatch? Creating a new array here implies that if another illustration was previously set with mouth shape and blink animation, it will be cleared